viam-cartographer.go

```diff
@@ -43,8 +49,25 @@ const (
 	defaultSensorValidationIntervalSec   = 1
 	parsePortMaxTimeoutSec               = 60
 	localhost0                           = "localhost:0"
+	DefaultCartoFacadeTimeout            = 5 * time.Second
 )
 
+var defaultCartoAlgoCfg = cartofacade.CartoAlgoConfig{
+	OptimizeOnStart:      false,
```

Comment:
confirm these are the correct defaults

Reply:
@JohnN193 & @nroede can y'all confirm these are the correct default config values for cartographer?

Reply:
yes these are correct and match our [docs](https://docs.viam.com/services/slam/cartographer/#config_params)(which we updated after the tuning spike)

Reply:
thank you!